content/applications/services/helpdesk/advanced/close_tickets.rst

```diff
@@ -1,54 +1,46 @@
 ======================================
-Allow Customers to Close their Tickets
+Allow customers to close their tickets
 ======================================
 
 Allowing customers to close their tickets gives them autonomy and minimize misunderstandings about
-when an issue is considered solved, or not. It makes communication and actions more efficient.
+when an issue is considered solved or not.
 
 Configure the feature
 =====================
 
-To configure the feature go to :menuselection:`Helpdesk --> Settings --> Helpdesk Teams --> Edit`
-and enable *Ticket closing*.
+To configure the feature, go to :menuselection:`Helpdesk --> Configuration --> Helpdesk Teams`,
+click on the team, and click :guilabel:`Edit` and enable :guilabel:`Ticket closing`.
 
-.. image:: close_tickets/closetickets1.png
-   :align: center
-   :alt: Ticket closing in Odoo Helpdesk
+.. image:: close_tickets/close-ticket-settings.png
+   :alt: Ticket closing feature in Odoo Helpdesk.
 
-In order to designate to which stage the ticket migrates to once it is closed, go to
-:menuselection:`Helpdesk --> Overview --> Tickets`.
+To designate which stage the ticket migrates to once it is closed, navigate to the ticket pipeline
+by going to :menuselection:`Helpdesk --> Overview` and clicking :guilabel:`Tickets` on the team's
+card.
 
-.. image:: close_tickets/closetickets2.png
-   :align: center
-   :alt: Ticket closing in Odoo Helpdesk
+There are two options: create a new Kanban stage or work with an existing one. For both scenarios,
+click the gear icon next to the stage name, select :guilabel:`Edit Stage`, and enable
+:guilabel:`Closing Stage`.
 
-You can either create a new Kanban stage or work with an existing one. For both scenarios, go to
-:menuselection:`Helpdesk --> Settings --> Edit Stage` and enable *Closing Stage*.
+If a closing stage is not specified, by default, the ticket is moved to the last stage. If more
+than one stage is set as a closing stage, the ticket is put in the first one.
```

Comment:
Be specific, especially when writing a new paragraph. Every paragraph should be able to stand on it's own without relying on surrounding context. Constantly remind the reader where they are and what to look for.

E.g. 
- `is moved to the last stage` --> the last stage, where?
- `is put in the first one` --> the first what?

This would also be better as a **note**.

```suggestion
.. note::
   If a closing stage is not specified, by default, the ticket is moved to the last stage in the
   kanban. If more than one stage is set as a closing stage, the ticket is placed in the first
   closing stage column.
```